lib/Doctrine/DBAL/ColumnCase.php

```diff
@@ -21,10 +25,13 @@
      */
     public const LOWER = 2;
```

Comment:
Are public constants still needed or their values could be moved directly to the methods which use them?

Reply:
Explained below.